app/src/main/res/values-de/strings-de.xml

```diff
@@ -1,255 +1,533 @@
-<resources xmlns:tools="http://schemas.android.com/tools" tools:ignore="MissingTranslation">
-
-    <string name="result_poster_img_des">Poster</string>
+    <!-- IS NOT NEEDED TO TRANSLATE AS THEY ARE ONLY USED FOR SCREEN READERS AND WONT SHOW UP TO NORMAL USERS -->
```

Comment:
you have removed a very important line: `<resources xmlns:tools="http://schemas.android.com/tools" tools:ignore="MissingTranslation">`